CONTRIBUTING.rst

```diff
@@ -0,0 +1,79 @@
+--------------------
+Contributing to PUDL
+--------------------
+
+Welcome! We're so glad you're interested in contributing to PUDL! We would love
+some help making PUDL data as complete as possible.
+
+.. _after-intro:
+
+.. IMPORTANT:: Already have a dataset in mind?
+
+   If you **need data that's not in PUDL** that we're missing in PUDL,
+   `open an issue <https://github.com/catalyst-cooperative/pudl/issues/new/choose>`__.
+
+   If you've **already written some code to wrangle a dataset**, find us at
+   `office hours <https://calend.ly/catalyst-cooperative/pudl-office-hours>`__ and we
+   can talk through next steps for how to get that into PUDL.
```

Comment:
I think we should frame this in a way that makes it clear that not every dataset is a good candidate for integration. Maybe 'find us at office hours and we can talk through next steps' is sufficient.